index.js

```diff
@@ -28,5 +30,7 @@ module.exports = function (file) {
     }
     this.emit('end');
   }
-  return through(ondata, onend);
+  return (path.extname(file) === '.json')
+    ? new stream.PassThrough()
```

Comment:
Could we just stick to `through` here? Wouldn't `through()` work since that is a pass through as well?